src/core/CCList.mli

```diff
@@ -435,33 +442,35 @@ val all_ok : ('a, 'err) result t -> ('a t, 'err) result
     @since 2.2 with labels *)
 
 val sorted_merge : cmp:('a -> 'a -> int) -> 'a list -> 'a list -> 'a list
-(** Merge elements from both sorted list. *)
+(** [sorted_merge ~cmp l1 l2] merges elements from both sorted list using
+    the given comparison function [cmp]. *)
 
 val sort_uniq : cmp:('a -> 'a -> int) -> 'a list -> 'a list
-(** Sort the list and remove duplicate elements. *)
+(** [sort_uniq ~cmp l] sorts the list [l] using the given comparison function [cmp]
+    and remove duplicate elements. *)
 
 val sorted_merge_uniq : cmp:('a -> 'a -> int) -> 'a list -> 'a list -> 'a list
-(** [sorted_merge_uniq l1 l2] merges the sorted lists [l1] and [l2] and
+(** [sorted_merge_uniq ~cmp l1 l2] merges the sorted lists [l1] and [l2] and
     removes duplicates.
     @since 0.10 *)
 
 val is_sorted : cmp:('a -> 'a -> int) -> 'a list -> bool
-(** [is_sorted l] returns [true] iff [l] is sorted (according to given order).
+(** [is_sorted ~cmp l] returns [true] iff [l] is sorted (according to given order).
     @param cmp the comparison function (default [Stdlib.compare]).
```

Comment:
`cmp` is a mandatory argument, `(default [Stdlib.compare])` should be removed.

Reply:
OK.